oaknational/Oak-Web-Application
# Feat/lesq 256/mathjax equations [LESQ-256]

## Description
- adds the [better react mathjax](https://github.com/fast-reflexes/better-react-mathjax) library to transform equations in quizzes
- wraps the quiz question list in `MathJaxContext` which provides the mathjax library plus a helper component
- wrap each quiz list item in `MathJax` helper component which converts any eligible mathjax syntax in its children into equations
- I've set the quiz questions to hide until the mathjax is loaded otherwise there's a flash of raw syntax on first load - it's quick but possible we'd want a loading state?
- added a global style on the mathjax element `mjx-container` to display `inline-grid` - the `MathJax` component exposes an `inline` flag but I couldn't get this to work consistently

## How to test

1. Go to https://deploy-preview-1918--oak-web-application.netlify.thenational.academy/teachers-2023/programmes/maths-secondary-ks4-higher/units/surds/lessons/identifying-square-factors-to-support-simplifying-surds
2. You should see the quiz questions rendering equations

## Screenshots

How it used to look:
<img width="404" alt="Screenshot 2023-09-20 at 15 30 12" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/74d5cba4-70b7-4172-aced-c7af6c08e8cc">


How it should now look:
<img width="401" alt="Screenshot 2023-09-21 at 08 31 56" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/d4c71510-90a6-4208-9a2c-cfe65771c730">

ACs:
- [x] I want mathjax compatible equations to render in quizzes
